torch/_inductor/codecache.py

```diff
@@ -809,10 +809,13 @@ def get_include_and_linking_paths(
             else:
                 macros = f"-D{macros}"
         if cuda:
-            if config.is_fbcode():
-                libs += ["cuda"]
```

Comment:
These lines are likely needed, and won't be covered by github tests.

Reply:
Thanks for taking a look @jansel, we haven't removed these for nvidia but we have conditionalised them out for ROCm only replacing with the equivalent rocm/hip libraries.

Full snippet
```
if cuda:
    if torch.version.hip is not None:
        libs += ["c10_hip", "torch_hip"]
    else:
        if config.is_fbcode():
            libs += ["cuda"]
        else:
            libs += ["c10_cuda", "cuda", "torch_cuda"]
```

Reply:
Got it, my mistake